mumble_backend/build_number_generator/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class BuildNumberGeneratorConfig(AppConfig):
+    default_auto_field = "django.db.models.BigAutoField"
+    name = "build_number_generator"
```

Comment:
It's not only a generator - it can also be used to query a build number for a given series + commit. So maybe better use
```suggestion
    name = "build_number_manager"
```
?